deploy-argocd.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+main() {
+    verify_permissions || exit $?
+    install_argocd
+    wait_for_pods_ready
+    port_forward_argocd
+    login_argocd
```

Comment:
These functions are not needed as ArgoCD is already installed on the EKS cluster.

Reply:
removed those functions